targets/targets.json

```diff
@@ -631,6 +631,17 @@
         "detect_code": ["0603"],
         "release_versions": ["5"]
     },
+    "EV_COG_AD3029LZ": {
+        "inherits": ["Target"],
+        "core": "Cortex-M3",
+        "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
+        "macros": ["__ADUCM3029__", "EV_COG_AD3029LZ"],
+        "extra_labels": ["Analog_Devices", "ADUCM302X", "ADUCM3029", "EV_COG_AD3029LZ", "FLASH_CMSIS_ALGO"],
```

Comment:
```
"EV_COG_AD3029LZ",
```
is not needed here, as it matches the name of the target exactly. The target is already a `label` so adding it again, with `extra_labels`, is redundant.